### ISSUE TITLE
Error failed to fetch http://deb.debian.org/debian/dists/jessie/InRelease

### ISSUE THREAD

**Issue by pir1981** [tianon/docker-brew-debian#57]
Hi,
since yesterday 14.12. I cannot build any of my docker build files FROM jessie:latest anymore (which were just fine previously), as they always exit with the below. It seems quite independent of the packages I try to apt-get. I checked the net which suggests looking into DNS, VPN, firewall ... but I have neither changed anything, nor am I running VPN or a firewall. 

Is it possible that the issue is due to [DSA-3733-1 apt -- security update](https://www.debian.org/security/2016/dsa-3733)
If so could you please update the jessie:latest image or help me understand how I can fix it myself?

```
Step 2 : RUN apt-get update     && apt-get install -y --no-install-recommends           bzip2           ca-certificates                 libffi-dev              libgdbm3 libssl-dev              libyaml-dev             procps          zlib1g-dev      && rm -rf /var/lib/apt/lists/*
 ---> Running in c88dd90c6d06
Err http://deb.debian.org jessie InRelease

Err http://security.debian.org jessie/updates InRelease

Err http://deb.debian.org jessie-updates InRelease

Err http://security.debian.org jessie/updates Release.gpg
  Could not resolve 'security.debian.org'
Err http://deb.debian.org jessie Release.gpg
  Could not resolve 'deb.debian.org'
Err http://deb.debian.org jessie-updates Release.gpg
  Could not resolve 'deb.debian.org'
Reading package lists...
W: Failed to fetch http://deb.debian.org/debian/dists/jessie/InRelease

W: Failed to fetch http://deb.debian.org/debian/dists/jessie-updates/InRelease

W: Failed to fetch http://security.debian.org/dists/jessie/updates/InRelease

W: Failed to fetch http://deb.debian.org/debian/dists/jessie/Release.gpg  Could not resolve 'deb.debian.org'

W: Failed to fetch http://security.debian.org/dists/jessie/updates/Release.gpg  Could not resolve 'security.debian.org'

W: Failed to fetch http://deb.debian.org/debian/dists/jessie-updates/Release.gpg  Could not resolve 'deb.debian.org'

W: Some index files failed to download. They have been ignored, or old ones used instead.
Reading package lists...
Building dependency tree...
Reading state information...
Package bzip2 is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'bzip2' has no installation candidate
E: Unable to locate package ca-certificates
E: Unable to locate package libffi-dev
E: Unable to locate package libgdbm3
E: Unable to locate package libssl-dev
E: Unable to locate package libyaml-dev
E: Unable to locate package zlib1g-dev
ERROR: Service 'ruby' failed to build: The command '/bin/sh -c apt-get update   && apt-get install -y --no-install-recommends           bzip2           ca-certificates libffi-dev               libgdbm3                libssl-dev              libyaml-dev             procps          zlib1g-dev      && rm -rf /var/lib/apt/lists/*' returned a non-zero code: 100
```

**Comment by yosifkit** [comment]
That really looks like a networking problem. "could not resolve" points to not getting a DNS response.  Can you ping an IP from a container?  What about a host like "google.com" or "security.debian.org"?  Perhaps you need to restart docker or the VM/machine it is running on?

**Comment by pir1981** [comment]
i did the following without any error messages (restarted fine and pulled images)

- `sudo /etc/init.d/docker restart`

- ` sudo docker pull debian:8.6`

- sudo powerdown -P now

- hardware restart

but as soon as I start building on the pulled image, I get the above errors. 
I did also try as @yosifkit suggested. It works perfectly fine directly from my machine, but it just returns with `ping: unknown host` when used from within a docker container. 
Yet the docker image is an nginx server with PHP and my pages running on that server are reachable from the outside without any issues. Any other suggestion for me to try?

Now remembering, I did have a "no local networks" error a day or two back, where I read that I have to delete a few unused networks `sudo docker network ls`. Could I have screwed something up? And if so any suggestions how to get it up and running again? It now has a bridge, host, none, data_default and various of my own build names.
